### DISCUSSION TITLE
Background page exploit

### DISCUSSION THREAD

**Discussion by axolotl8** [3kh0/ext-remover#1082] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I don't know if someone has already brought this up or not, but there is a method my friends and I at school discovered to bypass Lightspeed. Developer Mode needs to be unblocked. Go to chrome://extensions, and click on the background page of the extension you want to block. Then, click on Network. While in Network, click Disable Cache. Also, click on No throttling, and be sure to change that to Offline. Once that is finished, head to Settings, and scroll down. Click on the Disable Javascript box. DO NOT CLOSE THIS WINDOW (unless you want the extension to work again). @3kh0, please include this in your Ext Remover website.

**Comment by S-PScripts** [comment]
>  Developer Mode needs to be unblocked. 

what's even the point if there's so many better exploits requiring developer mode

**Reply by S-PScripts** [reply]
i guess so
also when i first saw the title i thought it was talking about how to change the wallpaper background

**Reply by axolotl8** [reply]
> lol

**Reply by axolotl8** [reply]
thx

**Comment by Inglan2** [comment]
you need devtools tho

**Reply by axolotl8** [reply]
Its the one with the load, pack and update extensions, not the one where you can unenroll.
